src/components/Recovered/Recovered.js

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import FLECHE from '../../assets/images/fleche.png';
+import './Recovered.css';
+import MonthRecoveredChart from './ChartRecoveredMonth';
+
+const RecoveredCases = () => {
+  const toDay = new Date();
+  const lastDay = `${toDay.getFullYear()}-${toDay.getMonth() + 1}-${toDay.getDate() - 1}`;
+  const previousDay = `${toDay.getFullYear()}-${toDay.getMonth() + 1}-${toDay.getDate() - 2}`;
+  let updateData = useSelector((state) => state.latest);
+  let dayBeforeData = useSelector((state) => state.dayBefore);
+  let growingRate = 1.2;
+  let totalRecovered = 20;
+  let newRecovered = 20;
+  let yesterdayTotalRecovered = 12;
+  let todayVsYesterday = 0.5;
+  if (Object.keys(updateData).length > 0) {
+    updateData = updateData.dates[lastDay].countries.Tunisia;
+    todayVsYesterday = Math.round(updateData.today_vs_yesterday_recovered);
+    yesterdayTotalRecovered = updateData.yesterday_recovered;
+    newRecovered = updateData.today_new_recovered;
+    totalRecovered = updateData.today_recovered;
+    if (Object.keys(dayBeforeData).length > 0) {
+      dayBeforeData = dayBeforeData.dates[previousDay].countries.Tunisia;
+      const newRecoveredDayBefore = dayBeforeData.today_new_recovered;
+      growingRate = (
+        (newRecovered - newRecoveredDayBefore) / Math.max(newRecoveredDayBefore, 1)
+      ) * 100;
+      growingRate = Math.floor(growingRate);
+    }
+  }
```

Comment:
Or you might want to make sure that the calculation of the growingRate it's correct. I think there are plenty of tests that you can implement. So please implement some, it's a good practice and very valuable in almost every company.